vendor/modules.txt

```diff
@@ -228,3 +256,6 @@ gopkg.in/yaml.v2
 # sigs.k8s.io/yaml v1.1.0
 ## explicit
 sigs.k8s.io/yaml
+# github.com/gogo/protobuf => github.com/gogo/protobuf v1.2.1
+# go.etcd.io/etcd/c/v3 => ./c
+# go.etcd.io/etcd/integration/v3 => ./SHOULD_NOT_BE_DEPENDENCY
```

Comment:
Interesting. How does this line get added to modules.txt? Is it added manually or do we have some way to make sure it gets added automatically?

Reply:
It's a consequence of: https://github.com/etcd-io/etcd/pull/12204/commits/053201df78b6dc034ca20101569c900d2948ebe9#diff-37aff102a57d3d7b797f152915a6dc16R54 in go.mod

```
replace (
	go.etcd.io/etcd/integration/v3 => ./SHOULD_NOT_BE_DEPENDENCY
)
```
A little bit a hack, but very explicit when the error happens.